app/views/404.html

```diff
@@ -0,0 +1,31 @@
+<!-- Page that will show when user goes to a page that doesn't exist (404 error) -->
+<style>
```

Comment:
Let's move these styles inline

Reply:
@AkashBhave We don't accept template styling, minor inline styling is acceptable, suggestion would be make changes in scss file and then it will automatically compile to css.

CC - @ManrajGrover